core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -464,7 +465,15 @@ public void invoke() {
     /**
      * Executable name for Docker Compose.
      */
-    private static final String COMPOSE_EXECUTABLE = "docker-compose";
```

Comment:
I would keep final and use something like `SystemUtils.IS_OS_WINDOWS ? "docker-compose.exe" : "docker-compose"`